### PR TITLE
Add description to Mars

### LINKED ISSUE #1: Add description of Mars
Can you add a description of Mars to planets.txt?

### PR DESCRIPTION
**Pull request by carschno**
Fixes #1 .

### CODE REVIEW

File: planets.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 Mars is a planet.
+It is rounbd.
```
Comment (ClaireDons) [inline planets.txt:2]: There is a typo on line 2.

**Review by ClaireDons — CHANGES_REQUESTED** [review]
Looks good, once we fix this typo, we can merge.

File: planets.txt (hunk 1 above)
Comment (carschno) [reply planets.txt]: Fixed!

**Review by ClaireDons — APPROVED** [review]
Looks good, you can merge now!

**Pull request merged**